src/Microsoft.ML.FastTree/TreeEnsemble/TreeEnsembleCombiner.cs

```diff
@@ -0,0 +1,116 @@
+﻿//------------------------------------------------------------------------------
+// <copyright company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//------------------------------------------------------------------------------
```

Comment:
you need this one
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.